code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -74,11 +74,11 @@
 		agent = preset_agent
 
 
-	scientist.assigned_role = "MODE"
+	scientist.assigned_role = "Abductor Scientist"
```

Comment:
Yes, I'm aware of that, but this will throw runtimes under certain conditions because `assigned_role` expects `MODE` if they're an antag or their station job role if they're not.

This is something that would require a much wider refactor that goes well beyond the scope of this PR, unfortuantely--it's not a quick wham-bam fix, unfortunately.